CHANGELOG.asciidoc

```diff
@@ -84,6 +140,7 @@ The following changes have been incorporated in devon4j:
 * https://github.com/oasp/oasp4j/issues/648[648]: Removed depreciated restaurant sample.[oasp4j]
 * https://github.com/oasp/oasp4j/issues/669[#669]: Updated Pagination support for Spring Data[oasp4j]
 * https://github.com/oasp/oasp4j/pull/671[#671]: Add support for hana as dbType.
+* https://github.com/devonfw/devon4j/pull/136[#136]: Create a migration guide from oasp4j 3.0.0 to devon4j 3.1.0
```

Comment:
Instead of this, could you add a link to the new guide to the text of the `3.0.0` release section as something like:

If you want to upgrade from oasp4j to devon4j, please consider the «https-link»[migration guide].